app/controllers/clarifying-questions/anything-else-answer.ts

```diff
@@ -32,6 +33,9 @@ function getAnythingElseAnswerPage(req: Request, res: Response, next: NextFuncti
 function postAnythingElseAnswerPage(updateAppealService: UpdateAppealService) {
   return async (req: Request, res: Response, next: NextFunction) => {
     try {
+      if (req.body['saveForLater']) {
```

Comment:
you are not saving answers if the appellant enters one.  Take a look at 'saveForLater.ts' controller and copy that logic.